lua/entities/terminator_nextbot_zambiemecha/shared.lua

```diff
@@ -0,0 +1,251 @@
+AddCSLuaFile()
+
+ENT.Base = "terminator_nextbot_zambie"
+DEFINE_BASECLASS( ENT.Base )
+ENT.PrintName = "Mecha Zombie"
+ENT.Spawnable = false
+list.Set( "NPC", "terminator_nextbot_zambiemecha", {
+    Name = "Mecha Zombie",
+    Class = "terminator_nextbot_zambiemecha",
+    Category = "Nextbot Zambies",
+} )
+
+ENT.SpawnHealth = 2250
+ENT.WalkSpeed = 70
+ENT.MoveSpeed = 180
+ENT.RunSpeed = 300
+ENT.AccelerationSpeed = 1000
+ENT.FistDamageMul = 1.5
+ENT.MyPhysicsMass = 200
+ENT.JumpHeight = 200
+
+ENT.DoMetallicDamage = true
+ENT.MetallicMoveSounds = true
+ENT.ReallyStrong = true
+ENT.ReallyHeavy = true
+
+ENT.HasBrains = true
+ENT.IsStupid = false
+ENT.IsFodder = false
+
+ENT.term_SoundPitchShift = -30
+ENT.term_SoundLevelShift = 10
+
+ENT.Mecha_LastShockwave = 0
+ENT.Mecha_ShockwaveCooldown = 5
+
+-- Mechanical sounds
+ENT.term_LoseEnemySound = "npc/scanner/scanner_talk1.wav"
+ENT.term_CallingSound = "npc/scanner/scanner_talk2.wav"
+ENT.term_CallingSmallSound = "npc/scanner/combat_scan5.wav"
+ENT.term_FindEnemySound = "npc/scanner/scanner_alert1.wav"
+ENT.term_AttackSound = "npc/scanner/scanner_combat1.wav"
+ENT.term_AngerSound = "npc/manhack/bat_away.wav"
+ENT.term_DamagedSound = "npc/scanner/scanner_pain1.wav"
+ENT.term_DieSound = "npc/scanner/scanner_explode_crash2.wav"
+ENT.term_JumpSound = "npc/scanner/scanner_nearmiss1.wav"
+
+ENT.IdleLoopingSounds = {
+    "npc/turret_floor/ping.wav",
+}
+ENT.AngryLoopingSounds = {
+    "npc/scanner/scanner_scan_loop2.wav",
+}

```

Comment:
These sounds feel like they were the first ones decided on maybe?
Sorry to be so harsh, i can do a pass on them but i'd love for you to give them another pass.
They just don't have that "powerful" feel to them.